.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches: # Trigger only on the master branch to avoid duplicate runs on PR branches
+      - master
+  pull_request: # Trigger All PRs
+
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ['2.4', '2.5', '2.6', '2.7', 'ruby-head']
+        include:
+          - ruby: 'ruby-head'
+            experimental: true # a hack to allow tests to fail for ruby-head, https://github.com/actions/toolkit/issues/399
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        # note: bundler-cache: true causes problems since we don't commit `Gemfile.lock`, etc.
+    - run: bundle install
+    - run: bundle exec srb tc
+    - run: bundle exec rspec || ${{ matrix.experimental == true }} # the eq forces a boolean instead of an empty string
```

Comment:
How did you figure out that you need the `==` in `matrix.experimental == true` for it to evaluate properly?

Reply:
With how GitHub Actions parsing works, if you just do `${{ matrix.experimental }}` instead, it will substitute in an empty string - which makes the resulting command `bundle exec rspec || `, which fails. [f227ce5](https://github.com/chanzuckerberg/sorbet-coerce/pull/62/commits/f227ce564b53a1ec120597960860e80eb29e09ac) has a CI output of this; note how it fails for all non-experimental items.